README.md

```diff
@@ -68,6 +68,10 @@ Sets timeout for installation task. Overrides the default timeout of 600s.
 
 Verbosity options for the installation (`debug` or `trace`). Writes verbose output to a log file on the host.
 
+#### `https_proxy` (Optional)
```

Comment:
@amolero-nr I thought we should add this to the environment variable section up above, Line 45 I believe

Reply:
Thanks again @Julien4218! Moved it up, see here: 84a087655c05d2ca7d3f3f3a329e2f68dd457c8d